scripts/download-file.ps1

```diff
@@ -0,0 +1,44 @@
+
+# This was modified from StackOverflow 
+# https://stackoverflow.com/questions/45574479/powershell-determine-new-url-of-a-permanently-moved-redirected-resource
+
+Param (
+    [Parameter(Mandatory, ValueFromPipeline)] [Uri] $Uri,
+    [string] $OutFile,
+    [int] $MaxRedirections = 50 # Use same default as [System.Net.HttpWebRequest]
+)
+
+process {
```

Comment:
This file is not used anymore, but was needed during testing because PowerShell can't redirect when it gets a 3xx status code.